acme/api/account.go

```diff
@@ -0,0 +1,207 @@
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/go-chi/chi"
+	"github.com/pkg/errors"
+	"github.com/smallstep/certificates/acme"
+	"github.com/smallstep/certificates/api"
+	"github.com/smallstep/certificates/logging"
+)
+
+// NewAccountRequest represents the payload for a new account request.
+type NewAccountRequest struct {
+	Contact            []string
+	OnlyReturnExisting bool
+}
+
+func validateContacts(cs []string) error {
+	for _, c := range cs {
+		if len(c) == 0 {
```

Comment:
personally, I don't really care (as you can tell since I'm barely doing any validation). I don't really see us using the contact info for anything. Maybe one of those things where we let users tell us if they would appreciate us being more strict?